README.md

```diff
@@ -1 +1,96 @@
-# chaos-actions
+# Github Action for Chaos Engineering in Kubernetes
+
```

Comment:
This README will be displayed in the marketplace please help me to improve this if I'm missing something. @ksatchit